pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -401,8 +401,8 @@ func SetupLibvirt() error {
 	}
 
 	if _, ok := os.LookupEnv("LIBVIRT_DEBUG_LOGS"); ok {
-		// see https://wiki.libvirt.org/page/DebugLogs for details
-		_, err = libvirtConf.WriteString("log_filters=\"3:remote 4:event 3:util.json 3:rpc 1:*\"\n")
+		// see https://libvirt.org/kbase/debuglogs.html for details
+		_, err = libvirtConf.WriteString("log_filters=\"3:remote 4:event 3:util.json 3:util.object 3:util.dbus 3:util.netlink 3:node_device 3:rpc 3:access 1:*\"\n")
```

Comment:
This seems to only change the debug settings. If that was intended could you reflect that in the PR description?

Reply:
Sorry, I'm not sure what you mean. The libvirt log filters are only used when the `debugLogs` label is applied to the VM, so yeah this change will only affect that scenario. Do you simply want me to be more explicit about his fact, eg. by saying "Improve libvirtd **debug** log filters", or are you referring to something else?

Reply:
Haha, yes that is pretty much it. Maybe also why the no-longer logged things are not helpful.

Reply:
That's explained in a lot of detail in the commit message. Is that somehow not specific enough? Should I replicate the same information in the PR as well? AFAICT only the title of a PR is kept in the git repo after merge.

Reply:
Yep, the PR should have an overall description in detail. Copying is good enough.

Reply:
Updated the commit message, the PR title and copied the explanation to the PR body.

Reply:
Thanks a lot.